app/models/force/preference.rb

```diff
@@ -49,13 +55,21 @@ class Preference < Force::ObjectBase
     def self.from_salesforce(attributes)
       preference = super
 
+      fields = preference.fields.salesforce
+
       # Special field conversion cases for preferences
-      if attributes.RecordType
-        preference.fields.salesforce['RecordType.DeveloperName'] = attributes.RecordType.DeveloperName
+      if fields.RecordType
```

Comment:
heads up, looks like this isn't quite working still.. 

no general apps show up in the table without filtering, and if I filter by general I get Live/work applications 

![image](https://user-images.githubusercontent.com/11825994/83553066-88db3e80-a4bf-11ea-9499-a24a38e2f62e.png)